golos.publication/golos.publication.cpp

```diff
@@ -91,7 +97,8 @@ void publication::create_message(structures::mssgid message_id,
                               std::string headermssg,
                               std::string bodymssg, std::string languagemssg,
                               std::vector<structures::tag> tags,
-                              std::string jsonmetadata) {
+                              std::string jsonmetadata,
+                              uint16_t curators_prcnt) {
```

Comment:
it's better to make this argument optional and set it's value to `min_curators_prcnt` if not passed

Reply:
I think @zxcat want to say use `std::optional<uint16_t>` for this parameter. So, you can detect that user passed value or not, and set `min_curators_prcnt` if not.